lib/typescript/render/BUILD

```diff
@@ -7,9 +7,9 @@ package(default_visibility = ["//visibility:public"])
 ts_web_library(
     name = "render",
     deps = [
-        "//frontend/assets",
+        "//frontend/assets", #TODO FIX ME
```

Comment:
It's not immediately obvious what needs to be fixed here (I get that this lib should not depend on assets, but I'd still spell it out)

Reply:
true that. I left it to myself as a reminder so that I can look into it (as of now I don't actually know why it is like that and what's the best action to take yet)